app/routes/file-upload.js

```diff
@@ -0,0 +1,73 @@
+const { BlobServiceClient } = require('@azure/storage-blob')
+const { v4: uuidv4 } = require('uuid')
+const storageConfig = require('../config/storage')
+const { checkFileExtension } = require('../utils/file-checks/extension-check')
+const { handleFileLimitExceeded } = require('../utils/file-checks/lenght-of-files-array-check')
+const { handleMessage } = require('../message')
+// const avConfig = require('../config/av-scan')
+// const { getAVToken } = require('../utils/av-scan/get-av-token')
+// const { sendToAvScan } = require('../utils/av-scan/send-to-AV')
```

Comment:
Just again, are lines 7-9 (and also lines 34 and 43) meant to be commented? If so just ignore this and mark it as resolved.